build/common.sh

```diff
@@ -674,7 +674,6 @@ function kube::build::sync_to_container() {
   # necessary.
   kube::build::rsync \
     --delete \
-    --filter='H /.git' \
     --filter='- /_tmp/' \
```

Comment:
sync_from_container filters `/_temp/` but not `/_tmp/` ...

Reply:
I'd like to focus on eliminating the rsync dance entirely.

... but I also don't think this will work very well unless rsync is taught to read .gitignore files directly, it seems pretty expensive to read .gitignore files recursively compared to the handful of exclusions here?